src/components/_shared/spinner.js

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import ScaleLoader from 'react-spinners/ScaleLoader';
+import { css } from '@emotion/core';
+import PropTypes from 'prop-types';
+
+import { COLORS } from 'enums';
+
+const Spinner = ({
+  isLoading = false,
+  containerHeight = 48,
```

Comment:
I know there aren't too many dynamic heights for the spinner but as best practice let's use propTypes to set a default container height. See below.